ticTacToe.ts

```diff
@@ -5,84 +5,150 @@ export type CellType = "" | "X" | "O";
 export type GameBoard = Array<Array<CellType>>;
 
 export class TicTacToe {
-  private gameBoard: GameBoard = [
-    ["", "", ""],
-    ["", "", ""],
-    ["", "", ""],
-  ];
-  private curentPlayer: CellType = "X";
   private winner: CellType | "None" = "";
   private countMoves = 0;
+  public gameBoard: GameBoard;
```

Comment:
I prefer to put publics on top and maybe event separate with line break for readability